src/backend/services/chat.py

```diff
@@ -679,6 +746,18 @@ def handle_stream_tool_calls_generation(
         )
     stream_event = StreamToolCallsGeneration(**event | {"tool_calls": tool_calls})
     stream_end_data["tool_calls"].extend(tool_calls)
+
+    # TODO: remove False condition to enable saving tool calls
+    if should_store and False:
+        save_tool_calls_message(
+            session,
+            tool_calls,
+            event.get("text", ""),
```

Comment:
this is the tool plan message right? 

since we store this as a regular chatbot message, FE should be provided some mechanism to know that this chatbot message is a tool plan message (since it is rendered in the UI differently)

maybe we add add a new role enum? https://github.com/cohere-ai/cohere-toolkit/blob/519a09634fee7448642aee89902f81b5f5d6a045/src/backend/schemas/chat.py#L17

Reply:
hmmm i wonder how much we want to deviate from the API 🤔 I agree that it would be better to have some to differentiate it, but this is how it's done in the API nowadays. 

What do you think about adding a `tool_plan` field and using it instead of `text`? It feels like it's a smaller change than adding another role

Reply:
ya that sounds fine 👍